parm/wflow/coldstart.yaml

```diff
@@ -102,7 +102,16 @@ metatask_run_ensemble:
     mem: '{% if global.DO_ENSEMBLE %}{%- for m in range(1, global.NUM_ENS_MEMBERS+1) -%}{{ "%03d "%m }}{%- endfor -%} {% else %}{{ "000"|string }}{% endif %}'
   task_make_ics_mem#mem#:
     <<: *default_task
-    command: '&LOAD_MODULES_RUN_TASK; "make_ics" "&JOBSdir;/JREGIONAL_MAKE_ICS"'
+    command: 
+      cyclestr:
+        value: 'source &USHdir;/load_modules_wflow.sh {{ user.MACHINE }} ; &SCRIPTSdir;/chgres_cube.py
```

Comment:
Honestly, I applied this change you recommended before but I was getting a weird error and reverted to work on other things - I'll try it again and see if I can find if Rocoto is setting that

Reply:
Hm, I'm getting a `lxml.etree.XMLSyntaxError: xmlParseEntityRef: no name, line 290, column 63` error during ./generate_FV3LAM_wflow.py, I'll look more into this

Reply:
I see. We should be able to have `lxml` provide the necessary escaping when constructing XML elements' text content. I'll take a look at this separately, but it might require a `uwtools` update, so we should probably resolve this comment and keep the semicolon for now. But we should not accept, if we have a choice, restrictions on use of bash syntax due to XML constraints.